<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFieldModel</class>
 <widget class="QWidget" name="frmFieldModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>域信息</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>286</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>基本属性</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>64</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>域英文名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>域中文名</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_datatype">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>116</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_cnname">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>90</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_enname">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>64</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>116</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据类型</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_fieldno">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>38</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>142</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据长度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>38</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>域号</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>220</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cb_keyword">
       <property name="text">
        <string>关键字</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_allownull">
       <property name="text">
        <string>可为空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_foreign">
       <property name="text">
        <string>是否外键</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="spin_length">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>145</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>显示类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_display_datatype">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>175</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>表号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_tableno">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>83</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_ok">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="group_ref">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>9</y>
     <width>201</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>引用属性</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>引用表名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="combo_ref_table"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>引用域名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="combo_ref_field"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="combo_display_column"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>显示列</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
